openlibrary/i18n/messages.po

```diff
@@ -0,0 +1,60 @@
+# Translation for Open Library.
+# Copyright (C) 2010 Internet Archive
+# This file is distributed under the same license as the Open Library
+# project.
+# Clonewayx <EMAIL@ADDRESS>, 2017.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Open Library VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2018-03-15 22:37+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: cs <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.5.3\n"
+"Language: id\n"
+
+#: account.py:347 account.py:382 account.py:422 password/reset_success.html:6
+#: verify.html:5 verify/activated.html:6 verify/success.html:6
+#, python-format
+msgid "Hi, %(user)s"
+msgstr "Halo %(user)s"
+
+#: account.py:348 account.py:383
+#, fuzzy, python-format
```

Comment:
```suggestion
#, python-format
```

`fuzzy` is used when the translation was auto-select but hasn't been approved. It will not be used in the UI! If this translation looks good, remove this `fuzzy` tag on this line.